src/main/java/com/gd/clinic/entities/Event.java

```diff
@@ -0,0 +1,39 @@
+package com.gd.clinic.entities;
+
+import lombok.*;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Events")
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Type(type = "uuid-char")
+    private UUID id;
+
+    @NonNull
+    @Type(type = "uuid-char")
+    private UUID prescriptionId;
+
+    @NonNull
+    @Type(type = "uuid-char")
+    private UUID patientId;
+
+    @NonNull
+    private OffsetDateTime dateTime;
+
+    @NonNull
+    private String status;
+
```

Comment:
event should have information about appointment it's related to

Reply:
That's right, that's why the class has a `prescriptionId` field so that such information may be joined in the database from the `Prescriptions` table (formerly known as appointment).  If the info should be linked in a different way, please let me know.